### DISCUSSION TITLE
Is it possible to add the board espressif_esp32s3_box to tinyuf2/ports/espressif/boards ?

### DISCUSSION THREAD

**Discussion by PaulskPt** [adafruit/tinyuf2#249] [General]
The board exists in adafruit/circuitpython/ports/espressif/boards.

![adafruit_tinyuf_ports_espressif_boards__can_the_espressif_esp32s3_box_be_added](https://user-images.githubusercontent.com/9107950/190450020-13658fba-22bb-49da-9ed0-b355b4de3ed7.png)

**Comment by hathach** [comment]
yeah, it is totally possible, I think I have an s3 box somewhere. Let me try to find it to add that board.

**Comment by hathach** [comment]
implemented by #251

**Comment by PaulskPt** [comment]
Thank you!

**Comment by PaulskPt** [comment]
Successfully built and flashed in an Ubuntu - Linux session. Here two images of the results:

![2022-10-17_15h00_S3BOXBOOT_after_successful_build_and_flash_of_tinyuf2_in_real_Ubuntu_Linux_session](https://user-images.githubusercontent.com/9107950/196200541-893e826c-1ead-47ec-babd-30767e14d5c5.png)


![2022-10-17_14h57_ESP32-S3-BOX_w_TinyUF2_](https://user-images.githubusercontent.com/9107950/196200577-f8413b4d-6271-4960-aa95-73de407f284d.jpg)

**Comment by PaulskPt** [comment]
Result after copying latest CircuitPython build .uf2 onto the boot disk:

![IMG_6995](https://user-images.githubusercontent.com/9107950/196214767-a1ecc9be-efb6-4635-9f9e-7eead7ef4992.jpg)

**Comment by hathach** [comment]
great picture, thank you for the update

**Comment by PaulskPt** [comment]
Thanks to your help I was able to upload a CPY script and modify it for the esp32-s3-box. You can watch the (first) result in a few short video I created last night. Here is the link to this video in a shared Dropbox folder: https://www.dropbox.com/sh/40bz462gduiv79m/AAD4aT0GGQoueV-YMywMQuNla?dl=0
